src/components/DayPicker.jsx

```diff
@@ -516,7 +535,7 @@ export default class DayPicker extends React.Component {
     this.setState({
       currentMonth: newMonth,
       monthTransition: null,
-      translationValue: 0,
+      translationValue: (this.props.isRTL && this.isHorizontal()) ? translationValueRTL : 0,
```

Comment:
when it's `0` on RTL the transition to the right is super fast and the transition to the left is cluttering a bit (not smooth)